src/hooks/useA11yNotification.js

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useMemo } from "react";
+
+export function useA11yNotification({
+  isUrgent = false,
+  isAriaLiveHandledOutside = false,
+  isIncludeActions = false,
+  ref
+}) {
+  // If notification include actions, focus need to pass move to the notification itself
+  useEffect(() => {
+    if (isIncludeActions && ref && ref.current) {
+      ref.current.focus();
```

Comment:
I am not sure if the focus handling should be part of monday-ui-react-core  hook or be handled separately in the monolith.
I thought that it will be better here because than developers could use this hook when developing custom notification components.